graphql/schema.js

```diff
@@ -0,0 +1,270 @@
+  name: 'Mutation',
+  description: 'Functions to mutate stuff',
+  fields: {
+    createQuestionnaire: {
+      type: Questionnaire,
+
+      args : {
+        title : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        description : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        theme : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        legalBasis : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        navigation : {
+          type : GraphQLBoolean
+        }
+      },
+
+      resolve(source, { title, description, theme, legalBasis, navigation }) {
+        return models.Questionnaire.create({
+          title,
+          description,
+          theme,
+          legalBasis,
+          navigation
+        });
+      }
+    },
+
+    createPage: {
+      type: Page,
+
+      args : {
+        title : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        description : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        questionnaireId : {
+          type : GraphQLID
+        }
+      },
+
+      resolve(source, { title, description, questionnaireId }) {
+        return models.Page.create({
+          title,
+          description,
+          QuestionnaireId : questionnaireId
+        });
+      }
+    },
+
+    createQuestion: {
+      type: Question,
+
+      args : {
+        title : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        description : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        guidance : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+        type : {
+          type : GraphQLString
+        },
+        mandatory : {
+          type : GraphQLBoolean
+        },
+        pageId : {
+          type : GraphQLID
+        }
+      },
+
+      resolve(source, { title, description, guidance, type, mandatory, pageId }) {
+        return models.Question.create({
+          title,
+          description,
+          guidance,
+          type,
+          mandatory,
+          PageId : pageId
+        });
+      }
+    },
+
+    createAnswer: {
+      type: Answer,
+
+      args : {
+        title : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+
+        description : {
+          type : GraphQLString
+        },
+
+        guidance : {
+          type : GraphQLString
+        },
+
+        label : {
+          type : GraphQLString
+        },
+
+        qCode : {
+          type : GraphQLString
+        },
+
+        type : {
+          type : new GraphQLNonNull(GraphQLString)
+        },
+
+        mandatory : {
+          type : GraphQLBoolean
+        },
+
+        questionId: {
+          type : GraphQLID
+        }
+      },
+
+      resolve(source, { title, description, guidance, label, qCode, type, mandatory, questionId }) {
+        return models.Answer.create({
```

Comment:
Just occurred to me that maybe we should introduce a "Service" layer in between the GraphQL API and Sequelize. It may be a bit overkill for now, but it would get rid of the direct dependency on Sequalize from the GraphQL api. Another benefit of doing this would be that additional Business logic e.g authentication could be done in this service layer. We don't need it yet for this PR though, just something to consider when we need it I suppose. 👍